spec/test_teacher_spec.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../classes/teacher'
+
+describe Teacher do
+  context 'When creating a Teacher' do
+    teacher = Teacher.new('Math', 22, 'Elias')
+    it "Should return 'Elias' as a teacher name" do
+      puts teacher.name
+      expect(teacher.name).to eq 'Elias'
+    end
+
+    it "Should return '22' as a teacher age" do
+      expect(teacher.age).to eq 22
+    end
+
+    it "Should return 'Math' as a teacher specialization" do
+      expect(teacher.specialization).to eq 'Math'
+    end
```

Comment:
- [ ] Great job so far 👏🏻 👏🏻 🥳  Just one thing miss, according to the project requirements, there should be unit tests for all the methods in all the classes, Kindly make sure the `can_use_services?` class method has (at least) an 'it' block 👇🏻 check your teacher,rb file

```ruby
  # Method can_use_services?
  def can_use_services?
    true
  end
```